how-to-guides/reproduce-run/scripts/reproduce_run.py

```diff
@@ -0,0 +1,126 @@
+import neptune
+import torch
+import torch.nn as nn
+from torchvision import datasets, transforms
+
+project_name = "common/showroom"
+
+######################
+# Step 1: Get run ID #
+######################
+# Fetch project
+project = neptune.init_project(
+    project=project_name, api_token=neptune.ANONYMOUS_API_TOKEN, mode="read-only"
+)
+
+# (Neptune) Fetch only inactive runs with tag "showcase-run"
+runs_table_df = project.fetch_runs_table(
+    state="inactive", tag=["showcase-run", "reproduce", "Basic script"]
+).to_pandas()
+
+# (Neptune) Stop project
+project.stop()
```

Comment:
```suggestion
# (Neptune) Fetch only inactive runs with tags "showcase-run", "reproduce", and "Basic script" in the project
with neptune.init_project(
    project=project_name, api_token=neptune.ANONYMOUS_API_TOKEN, mode="read-only"
) as project:
  runs_table_df = project.fetch_runs_table(
      state="inactive", tag=["showcase-run", "reproduce", "Basic script"]
  ).to_pandas()
```

Reply:
We have to stop the project because we are not using it anymore. I remember there is a slack thread that discussed this.

Reply:
Using it as a context manager automatically stops it